src/components/Banner.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import WOW from 'wowjs';
+
+class Banner extends Component { 
+    
+    componentDidMount() {
+        new WOW.WOW().init();
+    }
+
+    render() {
+        const DeveloperSide = {backgroundColor : "#fecfd0",
+                            backgroundImage: `url(/img/hero/ill-2.svg)`,
+                            backgroundSize: 'cover',
+                            backgroundPosition: 'right',
+                            backgroundRepeat: 'noRepeat',
+                            width: "100%"};
+        const DesignerSide = {backgroundColor : "#f0f0f0",
+                            backgroundImage: `url(/img/hero/ill-3.svg)`,
+                            backgroundSize: 'cover',
+                            backgroundPosition: 'right',
+                            backgroundRepeat: 'noRepeat',
+                            width: "100%"};
+        return (
+            <div className="jumbotron mb-0 bg-light">
+                <div className="container">
+                    {/* First Intro  */}
+                    <div className="row mb-5">
+                        <div className="col-md-7">
+                            <h1 className="display-4 text-uppercase">Hello Everyone</h1>
+                            <h1 className="display-5 text-uppercase mb-3">I'M Rattana Faengsut</h1>
```

Comment:
`'` should be escape

```suggestion
                            <h1 className="display-5 text-uppercase mb-3">I\'M Rattana Faengsut</h1>
```